[2021-06-29 14:33:58,746] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T14:13:06.368401+00:00 [queued]>
[2021-06-29 14:33:58,773] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T14:13:06.368401+00:00 [queued]>
[2021-06-29 14:33:58,773] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-29 14:33:58,773] {taskinstance.py:1068} INFO - Starting attempt 5 of 5
[2021-06-29 14:33:58,774] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-29 14:33:58,790] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): store_data> on 2021-06-29T14:13:06.368401+00:00
[2021-06-29 14:33:58,795] {standard_task_runner.py:52} INFO - Started process 23285 to run task
[2021-06-29 14:33:58,804] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'open_aq', 'store_data', '2021-06-29T14:13:06.368401+00:00', '--job-id', '603', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/open_aq.py', '--cfg-path', '/tmp/tmpvc6wzt2j', '--error-file', '/tmp/tmpeqii3olq']
[2021-06-29 14:33:58,805] {standard_task_runner.py:77} INFO - Job 603: Subtask store_data
[2021-06-29 14:33:58,895] {logging_mixin.py:104} INFO - Running <TaskInstance: open_aq.store_data 2021-06-29T14:13:06.368401+00:00 [running]> on host airflowvm
[2021-06-29 14:33:59,012] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=open_aq
AIRFLOW_CTX_TASK_ID=store_data
AIRFLOW_CTX_EXECUTION_DATE=2021-06-29T14:13:06.368401+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-29T14:13:06.368401+00:00
[2021-06-29 14:33:59,030] {base.py:69} INFO - Using connection to: id: ***_default. Host: localhost, Port: 5432, Schema: , Login: ***, Password: ***, extra: {'cursor': 'realdictcursor'}
[2021-06-29 14:33:59,041] {python.py:151} INFO - Done. Returned value was: None
[2021-06-29 14:33:59,056] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=open_aq, task_id=store_data, execution_date=20210629T141306, start_date=20210629T143358, end_date=20210629T143359
[2021-06-29 14:33:59,101] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-29 14:33:59,143] {local_task_job.py:151} INFO - Task exited with return code 0
